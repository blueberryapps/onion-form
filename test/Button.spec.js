/* eslint-disable react/no-multi-comp */

import React from 'react';
import Button from '../src/Button.react';
import { shallow } from 'enzyme';

describe('Button', () => {
  const wrapper = shallow(<Button disabled onClick={() => {}}>Send</Button>);

  it('should have children prop', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should have onClick in prop', () => {
    expect(typeof wrapper.prop('onClick')).toBe('function');
  });

  it('should have disabled in prop', () => {
    expect(wrapper.prop('disabled')).toBe(true);
  });
});
