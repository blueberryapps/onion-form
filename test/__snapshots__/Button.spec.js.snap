// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should have children prop 1`] = `
<button
  disabled={true}
  onClick={[Function]}
>
  Send
</button>
`;
